### PR TITLE
Cut gridblocks

### PR DESCRIPTION
**Pull request by jamesros161**
This change removes the GridBlock functionality from the Inspirations plugin all together.

### CODE REVIEW

File: includes/class-boldgrid-inspirations-inspiration.php [hunk 1]
```diff
@@ -223,27 +223,16 @@ public function add_hooks() {
 			$boldgrid_easy_attachment_preview_size = new Boldgrid_Inspirations_Easy_Attachment_Preview_Size();
 			$boldgrid_easy_attachment_preview_size->add_hooks();
 
-			// Asset Manager.
```
Comment (bwmarkle) [inline includes/class-boldgrid-inspirations-inspiration.php]: This class is still required.

File: includes/class-boldgrid-inspirations-asset-manager.php [hunk 2]
```diff
@@ -358,103 +358,6 @@ public function attach_asset( $params ) {
...
 	}
 
-	/**
-	 * When inserting a gridblock, download and attach the assets used within.
-	 * Then replace the empty 'url' with the url of the asset.
-	 * Example $boldgrid_asset_ids link below.
-	 *
-	 * @link http://pastebin.com/sP0kRdGp
-	 */
-	public function boldgrid_insert_attribute_assets( $boldgrid_asset_ids ) {
```
Comment (bwmarkle) [inline includes/class-boldgrid-inspirations-asset-manager.php]: You removed the method, but calls to it within the file still exist.

File: includes/class-boldgrid-inspirations-asset-manager.php [hunk 3]
```diff
@@ -122,37 +122,6 @@ public function __construct() {
...
-					'boldgrid_insert_attribute_assets'
-				) );
-
-			add_filter( 'boldgrid_gridblock_insert_dynamic_images',
-				array (
-					$this,
-					'boldgrid_gridblock_insert_dynamic_images'
-				) );
-
-			add_filter( 'delete_attachment', array( $this, 'delete_attachment' ) );
```
[the remark this answers is not in the document]
Comment (bwmarkle) [reply includes/class-boldgrid-inspirations-asset-manager.php]: Hi @jamesros161 you *resolved* this conversation, but you didn't reply or undo the delete.

[the remark this answers is not in the document]
Comment (jamesros161) [reply includes/class-boldgrid-inspirations-asset-manager.php]: Restored that method, but only left the delete_attachment filter

**Pull request merged**